core/templates/dev/head/pages/preferences/Preferences.js

```diff
@@ -17,11 +17,11 @@
  */
 
 oppia.controller('Preferences', [
-  '$scope', '$http', '$rootScope', '$modal', '$timeout', '$translate',
+  '$scope', '$http', '$rootScope', '$uibModal', '$timeout', '$translate',
   'AlertsService', 'UrlInterpolationService', 'UtilsService',
   'DASHBOARD_TYPE_CREATOR', 'DASHBOARD_TYPE_LEARNER',
   function(
-      $scope, $http, $rootScope, $modal, $timeout, $translate, AlertsService,
+      $scope, $http, $rootScope, $uibModal, $timeout, $translate, AlertsService,
```

Comment:
Ditto on `AlertsService` and the rest of the imports.

Reply:
Done